Sources/SwiftCurrent_SwiftUI/UIKitInterop/HostedWorkflowItem.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HostedWorkflowItem.swift
+//  SwiftCurrent_SwiftUI
```

Comment:
Thank you for adding the header.  I would like to keep this file in the SwiftUI module, but I realize that it probably makes more sense to be in the UIKit module since it really doesn't need anything from that module and you would only use it in a UIKit context.

I would like to leave it in this module UNTIL SwiftUI leaves beta and then we should consider moving it at that point.  That way if there are any additional tweaks we want to make to it, we have until that moment to tweak without version ramping.

This is more of a comment to @wwt/workflow-developers than to you @Tyler-Keith-Thompson .